lib/vrl/stdlib/benches/benches.rs

```diff
@@ -27,6 +27,8 @@ criterion_group!(
               format_number,
               format_timestamp,
               get_env_var,
+              // TODO: dynamic value
+              //get_host_ip,,
```

Comment:
Is this still planned TODO?

Reply:
Ah, yes. Currently `bench_function` doesn't support benchmarking these sorts of functions where the output is dynamic. I added to the list in https://github.com/timberio/vector/issues/6527